docs/blog/2019-02-20-introducing-use-static-query/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Introducing useStaticQuery
+date: 2019-02-20
+author: Sidhartha Chatterjee
+tags:
+  - hooks
+  - static query
+---
+
+React Hooks are _cool_. Besides simplifying your code and removing the need for a lot of boilerplate associated with classes in JavaScript (looking at you, `this`), they enable some serious shareability. They also make it possible to use state in functional components.
+
+You can probably tell that we've been super excited about [React Hooks][hooks-intro]. So when they finally landed in React 16.8, we figured it was time to give our very own `StaticQuery` component the hook treatment!
+
+### Say hello to useStaticQuery
+
+`useStaticQuery` is a hook that takes a GraphQL query and returns your data. That's it.
+
+**No more Render Props necessary to use a Static Query**
+
+This simplifies accessing data in your components and also keeps your component tree shallow.
+
+Let's quickly check out a basic example. Here's a typical Header component, first written using `StaticQuery` and then `useStaticQuery`.
+
+### Before
+
+```jsx
+import React from "react"
+import { StaticQuery, graphql } from "gatsby"
+
+const Header = () => {
+  return (
+    // highlight-next-line
+    <StaticQuery
+      query={graphql`
+        query {
+          site {
+            siteMetadata {
+              title
+            }
+          }
+        }
+      `}
+      // highlight-next-line
+      render={data => (
+        <header>
+          <h1>{data.site.siteMetadata.title}</h1>
+        </header>
+      )}
+    />
+  )
+}
+
+export default Header
+```
+
+### After
+
+```jsx
+import React from "react"
+import { useStaticQuery, graphql } from "gatsby"
+
+const Header = () => {
+  // highlight-next-line
+  const { site } = useStaticQuery(
+    graphql`
+      query {
+        site {
+          siteMetadata {
+            title
+          }
+        }
+      }
+    `
+  )
+
+  return (
+    // highlight-next-line
```

Comment:
I think the `return (` statement needs to be highlighted here...? (I may have gotten a line number wrong)

Reply:
I highlighted the next line since the value of the return changes but not the return statement. Not sure though. Here is what it looks like. 
![screenshot 2019-02-21 05 09 27](https://user-images.githubusercontent.com/7701981/53132803-e7b0c900-3596-11e9-9ff5-9a31c2d98323.png)
![screenshot 2019-02-21 05 09 33](https://user-images.githubusercontent.com/7701981/53132805-e8e1f600-3596-11e9-8055-e22f3c3dc07e.png)

Reply:
Oh yeah, you're totally right–I see it now! You can keep this as-is. :)